apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.Snippets/SpannerConnectionSnippets.cs

```diff
@@ -480,5 +481,14 @@ public void DataAdapter()
                 // End sample
             });
         }
+
+        [Fact]
+        public void DependencyInjection()
+        {
+            IServiceCollection services = new ServiceCollection();
+            // Sample: DependencyInjection
+            services.AddSpannerConnection("SpannerDb");
```

Comment:
Doesn't this fail? Where is the configuration to pull "SpannerDb" from, or what am I missing?

Reply:
It doesn't fail because we never ask for the connection. (It's only resolved that that point.)